wdl/MainVcfQc.wdl

```diff
@@ -0,0 +1,843 @@
+  RuntimeAttr runtime_override = select_first([runtime_attr_override, runtime_default])
+  runtime {
+    memory: "~{select_first([runtime_override.mem_gb, runtime_default.mem_gb])} GB"
+    disks: "local-disk ~{select_first([runtime_override.disk_gb, runtime_default.disk_gb])} HDD"
+    cpu: select_first([runtime_override.cpu_cores, runtime_default.cpu_cores])
+    preemptible: select_first([runtime_override.preemptible_tries, runtime_default.preemptible_tries])
+    maxRetries: select_first([runtime_override.max_retries, runtime_default.max_retries])
+    docker: sv_base_mini_docker
+    bootDiskSizeGb: select_first([runtime_override.boot_disk_gb, runtime_default.boot_disk_gb])
+  }
+
+  command <<<
+    # Create final output directory
+    mkdir "~{tar_folder_name}"
+
+    while read tarball_path; do
+      tar -xzvf "$tarball_path" --directory ~{tar_folder_name}/
+    done < ~{write_lines(in_tarballs)}
+
+    # Compress final output directory
+    tar -czvf "~{outfile_name}" "~{tar_folder_name}"
+  >>>
+
+  output {
+    File vid_lists = outfile_name
+  }
+}
+
+
+# Plot external benchmarking results
+task PlotQcExternalBenchmarking {
+  input {
+    File benchmarking_tarball
+    String prefix
+    String comparator
+    String sv_pipeline_qc_docker
+    RuntimeAttr? runtime_attr_override
+  }
+  RuntimeAttr runtime_default = object {
+    mem_gb: 3.75,
+    disk_gb: 20,
+    cpu_cores: 1,
+    preemptible_tries: 1,
+    max_retries: 1,
+    boot_disk_gb: 10
+  }
+  RuntimeAttr runtime_override = select_first([runtime_attr_override, runtime_default])
+  runtime {
+    memory: "~{select_first([runtime_override.mem_gb, runtime_default.mem_gb])} GiB"
+    disks: "local-disk ~{select_first([runtime_override.disk_gb, runtime_default.disk_gb])} HDD"
+    cpu: select_first([runtime_override.cpu_cores, runtime_default.cpu_cores])
+    preemptible: select_first([runtime_override.preemptible_tries, runtime_default.preemptible_tries])
+    maxRetries: select_first([runtime_override.max_retries, runtime_default.max_retries])
+    docker: sv_pipeline_qc_docker
+    bootDiskSizeGb: select_first([runtime_override.boot_disk_gb, runtime_default.boot_disk_gb])
+  }
+
+  command <<<  
+    set -eu -o pipefail
+      
+    # Plot benchmarking stats
+    /opt/sv-pipeline/scripts/vcf_qc/plotQC.external_benchmarking.helper.sh \
+      ~{benchmarking_tarball} \
+      ~{comparator}
+
+    # Prep outputs
+    tar -czvf ~{prefix}.collectQC_benchmarking_~{comparator}_output.wPlots.tar.gz \
+    collectQC_benchmarking_~{comparator}_output
+  >>>
+
+  output {
+    File tarball_wPlots = "~{prefix}.collectQC_benchmarking_~{comparator}_output.wPlots.tar.gz"
+  }
+}
+
+
+# Plot per-sample stats
+task PlotQcPerSample {
+  input {
+    File vcf_stats
+    File samples_list
+    File per_sample_tarball
+    String prefix
+    String sv_pipeline_qc_docker
+    RuntimeAttr? runtime_attr_override
+  }
+  RuntimeAttr runtime_default = object {
+    mem_gb: 7.75,
+    disk_gb: 50,
```

Comment:
Special attn to the removal of dynamic resource allocation for this task because the settings are higher than typical